### ISSUE TITLE
Enhance custom serializer

### ISSUE THREAD

**Issue by jh-RLI** [rl-institut/WAM_APP_FRED#7]
First collection of ideas on how to develope a  WAM-App-Function that returns geodata and attributes from a datasource(like postgreSQL) that holds a lot of data records. 
Since we use a custom serializer the functionality could be included by enhancing the serializer.

Main-topics are:
- Map performance on a client browser
- WebPage loading/response time on first entry
- response time after user interactions(move map, click map-feature, filter features: any interactions that trigger a communication between server and database and a html response from server to client)
- Implement well performing filters, etc.

**Comment by jh-RLI** [comment]
How others did this:
[Uber](https://eng.uber.com/maps-metrics-computation/)
[MS-Azure](https://azure.microsoft.com/de-de/resources/videos/azure-friday-visualizing-data-with-azure-maps-web-map-control/)

**Comment by jh-RLI** [comment]
The Python [multiprocessing](https://docs.python.org/3.5/library/multiprocessing.html?highlight=process) could help to query the database more efficient. @Bachibouzouk since you want to learn about this in the future maybe you can share your experiences here.

**Comment by nesnoj** [comment]
@4lm and I have put some time into maps+serialization in our app, if you're interested we could talk about it..

By the way: where's your serializer located?

**Comment by 4lm** [comment]
Yes, try the simple stuff first:

- as @nesnoj mentioned Python [multiprocessing](https://docs.python.org/3.7/library/multiprocessing.html?highlight=process) is a super simple way to offload work from the process Django is running in. I tried it this weekend for a private project of mine and it took me literally 3 minutes to implement.
- "WebPage loading/response time on first entry", in our case most of time was taking up by parsing of geoJSON data and not the data transfer, we found a simple solution caching (we use it in RAM, but could be made static via disk caching, we reduced the load time from 22sec. to 2 sec.): [Django Caching](https://docs.djangoproject.com/en/2.2/topics/cache/).
